DatabaseConnector.pom.xml

```diff
@@ -7,23 +7,23 @@
     <parent>
         <groupId>com.mendix.databaseconnector</groupId>
         <artifactId>databaseconnector-main</artifactId>
-        <version>3.1.0</version>
+        <version>4.0.0</version>
         <relativePath>.</relativePath>
     </parent>
 
     <artifactId>databaseconnector</artifactId>
-    <version>3.1.0</version>
+    <version>4.0.0</version>
 
     <dependencies>
         <dependency>
             <groupId>com.zaxxer</groupId>
             <artifactId>HikariCP</artifactId>
-            <version>2.6.1</version>
+            <version>3.4.5</version>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>1.7.21</version>
+            <version>1.7.25</version>
```

Comment:
I believe Mx 8.12.x uses 1.7.28. And there is 1.7.30 available as well. Is there a reason why you picked .25 instead?

Reply:
I based it on version mentioned in `runtime/project/Dependencies.scala`. In MX 8.12.4 the version is 1.7.25

Reply:
Fixed